CODING-GUIDE.md

```diff
@@ -145,3 +145,10 @@ module RuleEngine =
   ```
 
   Failing to do so may result in conflicting generators or unexpected behaviour.
+
+## ReScript
+
+- we are moving our cs to tailwind:
```

Comment:
is this supposed to be 'css' or does cs stand for something?

Reply:
typo, thanks